targets/default.nix

```diff
@@ -13,6 +13,7 @@
 }:
 lib.foldr lib.recursiveUpdate {} [
   (import ./nvidia-jetson-orin.nix {inherit self lib nixpkgs nixos-generators microvm jetpack-nixos;})
+  (import ./nvidia-jetson-orin-nx.nix {inherit self lib nixpkgs nixos-generators microvm jetpack-nixos;})
```

Comment:
See comment https://github.com/tiiuae/ghaf/pull/209/files#r1255589951 for the context:

Could we do instead:
```
  (import ./nvidia-jetson-orin.nix {inherit self lib nixpkgs nixos-generators microvm jetpack-nixos nvidia-jetson-orin-agx;})
  (import ./nvidia-jetson-orin.nix {inherit self lib nixpkgs nixos-generators microvm jetpack-nixos nvidia-jetson-orin-nx;})
```
See passing arguments with `import` at https://nixos.org/guides/nix-pills/functions-and-imports.html#idm140737320462832